server.go

```diff
@@ -610,6 +672,147 @@ func (s *server) WaitForShutdown() {
 	s.wg.Wait()
 }
 
+// configurePortForwarding attempts to set up port forwarding for the diffrent
+// ports that the server will be listening on.
+//
+// NOTE: This should only be used when using some kind of NAT traversal to
+// automatically set up forwarding rules.
+func (s *server) configurePortForwarding(ports ...uint16) error {
+	ip, err := s.natTraversal.ExternalIP()
+	if err != nil {
+		return err
+	}
+	s.lastIP = ip
```

Comment:
should probably be called `lastDetectedIP`, to not confuse with IP set in `ExternalIP`

Reply:
Fixed.